engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -5164,6 +5169,17 @@ struct fel_rush_t : public demon_hunter_attack_t
 
     p()->buff.unbound_chaos->expire();
 
+    if (p()->talent.havoc.dash_of_chaos->ok())
+    {
+      p()->buff.dash_of_chaos->trigger();
+    }
+
+    // Dash of chaos incurs 1 second lockout on Felrush
+    if ( p()->talent.havoc.dash_of_chaos && !p()->buff.dash_of_chaos->up() )
```

Comment:
`p()->talent.havoc.dash_of_chaos` needs to be `p()->talent.havoc.dash_of_chaos->ok()`

Reply:
this line has been removed in most recent commit